rego/lib/fugue/regula.rego

```diff
@@ -121,7 +121,7 @@ rule_resource_result(rule, judgement) = ret {
     "rule_message": judgement.message,
     "rule_result": result_string(judgement),
     "rule_name": rule["package"],
-    "platform": rule.input_type,
+    "platform": rule.input_type,  # TODO: Does the naming here still make sense?
```

Comment:
I think you're right that this value doesn't match this name. What do you think about having three values for `platform`:
* `terraform`
* `cloudformation`
* `runtime`

and map the input types back to those:

```
platform := {
  "tf": "terraform",
  "tf_plan": "terraform",
  "tf_runtime": "runtime",
  "cfn": "cloudformation"
}

input_type_to_platform(input_type) = ret {
  ret = platform[input_type]
} else = ret {
  ret = input_type
}
```

Reply:
@jason-fugue @curtis-fugue I'd suggest updating the field name to "input_type" so that we're actually consistent across the board. This is probably one of the last chances we'll have to do so.